Lib/test/libregrtest/main.py

```diff
@@ -673,8 +678,11 @@ def _execute_python(self, cmd, environ):
 
         cmd_text = shlex.join(cmd)
         try:
-            print(f"+ {cmd_text}", flush=True)
+            # Android and iOS run tests in embedded mode.
+            if not cmd[0]:
+                raise ValueError("No Python executable is present")
```

Comment:
I'm not sure I see how this helps - won't this just terminate the test suite?

Reply:
Ah - that was the piece I was missing. I see now that this shortcuts the attempt to call the subprocess, but flags that a subprocess call *should* have been used, which is as good as we're going to get under these circumstances.